README.md

```diff
@@ -18,3 +19,15 @@ Start development
 Build Project
 
 `npm run build`
+
+### Deployment
+
+Install Firebase command line tools
+
+`npm install -g firebase-tools`
```

Comment:
lets try to install it only locally? without the -g. Normally it should simply work as npm adds the .bin folder automatically to the path.